US-CBP/cbp-theme
# Issue271

#### What's this PR do?
Replicates the dynamic application of class names needed to match the position changes of the CBP-Theme's .app-header with that of the App Directory's.

#### Where should the reviewer start?
By looking at the differences in the images provided in this issue's details.

#### How should this be manually tested?
Begin by inspecting the changes that occur in the class names for the <div class="app-header"> when scrolling down from the very top of pages for both the app directory and cbp theme indexes.

Next, check to see why the script from the 'app-navigation.js' file had to be manually added to the CBP Theme's index instead of being grabbed from /cbp-theme.browser.bundle.umd.js .
(looks to possibly be a bundling issue?)

#### Any background context you want to provide?
n/a
#### What are the relevant tickets?
n/a
#### Screenshots (if appropriate)
all are in this issue's details

#### Questions:
- Is there a blog post?
n/a
- Does the knowledge base need an update?
not sure
